src/views/states/list/StatesList.tsx

```diff
@@ -101,22 +101,30 @@ const StatesList: FC = () => {
             />
           </div>
 
-          <Table
-            cells={activeColumns}
-            rows={paginatedData}
-            gridBreakPoint={TableGridBreakpoint.none}
-            role="presentation"
-          >
-            <TableHeader />
-            {paginatedData.map((nns, index) => (
-              <StateRow
-                key={nns?.metadata?.name}
-                obj={nns}
-                activeColumnIDs={new Set(activeColumns.map(({ id }) => id))}
-                rowData={{ rowIndex: index, selectedFilters }}
-              />
-            ))}
-          </Table>
+          {filteredData.length > 0 ? (
+            <Table
+              cells={activeColumns}
+              rows={paginatedData}
+              gridBreakPoint={TableGridBreakpoint.none}
+              role="presentation"
+            >
+              <TableHeader />
+              {paginatedData.map((nns, index) => (
```

Comment:
I'd avoid using variable names like `nns`. It does not say anything. I'd maybe better use `nnstate` at least.

Reply:
okay